tests/test_client.py

```diff
@@ -4,7 +4,7 @@
 
 
 def test_client_init():
-    """ Test the client initialization. """
+    """Test the client initialization."""
```

Comment:
May want to consider moving code reformatting to a PR before this PR.  That will reduce the churn in this PR.